css/style.css

```diff
@@ -1194,3 +1244,112 @@ h2 a {
 #mw-list ul li {
  margin-left: -20px;
 }
+
+#blog-doc:has(> h1#express-blog), 
+#blog-doc:has(> h1#write-a-blog-post) {
+  min-height: 300px;
+  overflow-y: scroll;

```

Comment:
```suggestion
```

Reply:
Thanks and accepted. I do not see this scrollbar on my end + DO not want it there.
Added b/c I was having problems scrolling on mobile, but I can't seem to reproduce that issue now, so it's moot.